--- gcc-6.4.0/libgo/go/syscall/libcall_linux.go.old	2016-02-03 15:58:02.419872000 -0600
+++ gcc-6.4.0/libgo/go/syscall/libcall_linux.go	2018-06-22 00:56:31.478280552 -0500
@@ -355,19 +355,19 @@
 //sys	Setxattr(path string, attr string, data []byte, flags int) (err error)
 //setxattr(path *byte, name *byte, value *byte, size Size_t, flags _C_int) _C_int
 
-//sys	splice(rfd int, roff *_loff_t, wfd int, woff *_loff_t, len int, flags int) (n int64, err error)
-//splice(rfd _C_int, roff *_loff_t, wfd _C_int, woff *_loff_t, len Size_t, flags _C_uint) Ssize_t
+//sys	splice(rfd int, roff *_off_t, wfd int, woff *_off_t, len int, flags int) (n int64, err error)
+//splice(rfd _C_int, roff *_off_t, wfd _C_int, woff *_off_t, len Size_t, flags _C_uint) Ssize_t
 func Splice(rfd int, roff *int64, wfd int, woff *int64, len int, flags int) (n int64, err error) {
-	var lroff _loff_t
-	var plroff *_loff_t
+	var lroff _off_t
+	var plroff *_off_t
 	if roff != nil {
-		lroff = _loff_t(*roff)
+		lroff = _off_t(*roff)
 		plroff = &lroff
 	}
-	var lwoff _loff_t
-	var plwoff *_loff_t
+	var lwoff _off_t
+	var plwoff *_off_t
 	if woff != nil {
-		lwoff = _loff_t(*woff)
+		lwoff = _off_t(*woff)
 		plwoff = &lwoff
 	}
 	n, err = splice(rfd, plroff, wfd, plwoff, len, flags)
