--- gcc-8.3.0/libgo/mksigtab.sh.old	2017-11-22 00:27:29.000000000 +0000
+++ gcc-8.3.0/libgo/mksigtab.sh	2019-03-02 22:01:17.530000000 +0000
@@ -82,7 +82,7 @@
 checksig _SIGEMT     '{_SigThrow, "SIGEMT: emulate instruction executed"}'
 checksig _SIGINFO    '{_SigNotify, "SIGINFO: status request from keyboard"}'
 checksig _SIGTHR     '{_SigNotify, "SIGTHR: reserved"}'
-checksig _SIGPOLL    '{_SigNotify, "SIGPOLL: pollable event occurred"}'
+#checksig _SIGPOLL    '{_SigNotify, "SIGPOLL: pollable event occurred"}'
 checksig _SIGWAITING '{_SigNotify, "SIGWAITING: reserved signal no longer used by"}'
 checksig _SIGLWP     '{_SigNotify, "SIGLWP: reserved signal no longer used by"}'
 checksig _SIGFREEZE  '{_SigNotify, "SIGFREEZE: special signal used by CPR"}'
@@ -94,10 +94,12 @@
 
 # Special handling of signals 32 and 33 on GNU/Linux systems,
 # because they are special to glibc.
+# Signal 34 is additionally special to Linux systems with musl.
 if test "${GOOS}" = "linux"; then
-    SIGLIST=$SIGLIST"_32__33_"
+    SIGLIST=$SIGLIST"_32__33__34_"
     echo '	32: {_SigSetStack + _SigUnblock, "signal 32"}, /* SIGCANCEL; see issue 6997 */'
     echo '	33: {_SigSetStack + _SigUnblock, "signal 33"}, /* SIGSETXID; see issues 3871, 9400, 12498 */'
+    echo '	34: {_SigSetStack + _SigUnblock, "signal 34"}, /* see issue 30062 */'
 fi
 
 if test "${GOOS}" = "aix"; then
